packages/functions/src/dedup.ts

```diff
@@ -286,7 +297,7 @@ function dedupMaterials(document: Document): void {
 		}
 	}
 
-	logger.debug(`${NAME}: Found ${duplicates.size} duplicates among ${materials.length} materials.`);
+	logger.debug(`${NAME}: Found ${duplicates.size} duplicates among ${materials.length} materials. (Skipped ${keptMaterials} materials)`);
```

Comment:
I'd rather not track track and log reasons for why things were not merged, they cannot be merged, that is enough. The `dedup` function is used internally elsewhere so this is potentially noisy.

We may need to extend the same behavior to more object types – `mesh.weights` for example — so just want to keep this simple.

Reply:
Sure no problem. Would you say adding a tiny marker (e.g. postfix it with `*` if things were skipped) would be ok or should it just stay the way it was?

Reply:
I would rather not, a message equivalent to "at least one material was animated" or "N materials were animated" is not really actionable. I somewhat regret logging any counts here at all.

Reply:
I removed the count and modified the log as suggested below to `Merged 3 or 20 materials`